Suggest a new file naming for migrations (#432)
directus/docs: content/configuration/migrations.md

| ----------------- | ------------------------------------ | -------------- |
| `MIGRATIONS_PATH` | Where custom migrations are located. | `./migrations` |

> The file name follows the following structure `[identifier]-[name].js`, where `identifier` should not follow `YYYYMMDD[A-Z]` format, otherwise it may conflict with Directus intern migrations. For example, you can name your migrations like `001-initial-migration.js`.

Reviewer: ```suggestion
The file name follows the structure `[identifier]-[name].js`, where `identifier` should **not** follow the `YYYYMMDD[A-Z]` format, as this may conflict with Directus internal migrations. For example, you can name your migrations like `001-initial-migration.js`.
```
